subprojects/test-runner/instrumentation-tests/src/test/kotlin/com/avito/instrumentation/internal/finalizer/action/WriteTaskVerdictActionTest.kt

```diff
@@ -106,16 +106,46 @@ internal class WriteTaskVerdictActionTest {
                 ),
                 lostTest
             ),
-            failedTests = emptyList(),
-            lostTests = setOf(lostTest)
+            unsuppressedFailedTests = emptyList(),
+            notReportedTests = setOf(lostTest)
         )
         val action = createWriteTaskVerdictAction(tempDir)
         action.action(verdict)
 
         val verdictContent = readFile(tempDir)
 
-        assertThat(verdictContent).contains("Failed. There are 1 not reported tests")
-        assertThat(verdictContent).contains("com.test.Test2.test API22 LOST")
+        assertThat(verdictContent).contains("Not reported tests: 1")
+        assertThat(verdictContent).contains("com.test.Test2.test API22 NOT REPORTED")
+    }
+
+    @Test
+    fun `file - contains lost message without suppressed tests - verdict is failure`(@TempDir tempDir: File) {
```

Comment:
I didn't get the naming pattern.
Is "verdict is failure" a condition? In this case it doesn't explain it.

https://avito-tech.github.io/avito-android/contributing/Testing/#naming-conventions

Reply:
yes it is. misleading test name. will fix it